README.md

```diff
@@ -134,6 +134,23 @@ The followup screen can be skipped for promoters, so they are taken straight to
 wootric.shouldSkipFollowupScreenForPromoters(true);
 ```
 
+### Color customization (smartphones only)
+Colors can be customized for the smartphone's version.
+
+```java
+// Changes background color and text buttons color for the survey
+wootric.setSurveyColor(R.color.survey_color);
```

Comment:
Why do we have this public methods if modifying `colors.xml` will suffice to customize the survey ? If everything goes into that file I think this should be private